SETUP.md

```diff
@@ -106,6 +106,7 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
       <summary>Troubleshoot: pdftk errors</summary>
 
       * If it complains about pdftk, removing https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb from the above command seems to not have serious side effects (it will cause `PDFMergerTest` to fail). It may be a new URL is needed in the dependency list, see https://leancrew.com/all-this/2017/01/pdftk/
+      * pdftk can also be installed via gem: `sudo gem install pdftk`
```

Comment:
I'd rather not recommend installing gems globally with sudo if alternatives are available; would adding `pdftk` to our Gemfile suffice to ensure that this gets installed locally?

Reply:
Cool, I'll add it to the Gemfile. Should I remove pdftk from the brew install command entirely, or still have folks try that and remove it if it fails?

Reply:
I'm not 100% familiar with pdftk itself, but I'd say if installing just from Gemfile without brew at all works, then that's the way to go!